### PR TITLE
Milestone 2: Add API to the code

### PR DESCRIPTION
**Pull request by carloshs1994**
# In this milestone:

🌵 Create a new game by using the API. ({result: 'Game with ID: nUT3TlYnotX0jWAN8AyD added.'})
🌵 Implement the "Refresh" button (receiving data from the API and parsing the JSON).
🌵 Implement the form "Submit" button (sending data to the API).
🌵 Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -22,25 +22,25 @@ function addToLocalStorage(scores) {
   localStorage.setItem('scores', JSON.stringify(scores));
 }
 
-function appendScoresToList() {
+function appendScoresToList(scores) {
```
Comment (mateo951) [inline src/index.js]: You have a few old-school functions on this js file. Kindly make sure to use arrow functions instead to meet the project requirements 🙌

File: src/scores.js [hunk 2]
```diff
@@ -1,14 +1,43 @@
...
+  });
+  const json = await sendInfo.json();
+  return json;
+};
+
+const getScore = async () => {
+  const getInfo = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/nUT3TlYnotX0jWAN8AyD/scores/');
+  const json = await getInfo.json();
+  return json;
+};
```
Comment (mateo951) [inline src/scores.js:20]: New data is properly being submitted to the API and after the refresh button is clicked, new info is rendered as expected 🙌👌 Fantastic job!!

**Review by mateo951 — CHANGES_REQUESTED** [review]
###  Hi @carloshs1994, 

📝👌 **Your project highlights** 🔥😎 
- General Requirements 👌
- Your app works as intended 🔥🔥🔥

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: src/index.js (hunk 1 above)
Comment (carloshs1994) [reply src/index.js]: Thanks for the feedback @mateo951, I'll definitely change that. 👌🏻

File: src/scores.js (hunk 2 above)
Comment (carloshs1994) [reply src/scores.js:20]: # Thanks!

**Review by er-contreras — APPROVED** [review]
Hi @carloshs1994 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**